datalad_registry/datasets.py

```diff
@@ -32,3 +36,38 @@ def datasets():  # No type hints due to mypy#7187.
         return jsonify({"next": pg_n,
                         "previous": pg_p,
                         "ds_ids": [i.ds_id for i in r.items]})
+
+
+@bp.route("urls/<string:url_encoded>", methods=["GET", "PATCH"])
+def urls(url_encoded: str) -> Any:
+    try:
+        url = url_decode(url_encoded)
+    except InvalidURL:
+        return jsonify(message="Invalid encoded URL"), 400
+
+    result = db.session.query(URL).filter_by(url=url)
+    row_known = result.first()
+
+    if request.method == "GET":
+        lgr.info("Checking status of registering %s as URL", url)
+        resp: Any = {"url": url}
```

Comment:
why not 
```suggestion
        resp: dict = {"url": url}
```
?

Reply:
I was just copying the code from [here](https://github.com/datalad/datalad-registry/blob/7669bf4bd6597685119747e1b3025ee14d1f7297/datalad_registry/dataset_urls.py#L53).

Reply:
well, sooner or later we better tighten it up here and there

Reply:
Fixed.